docs/ldml/tr35-personNames.md

```diff
@@ -746,35 +755,43 @@ The following process is used to produce initials when they are not supplied by
 
-Some writing systems require spaces (or some other non-letters) to separate words. For example, [Hayao Miyazaki](https://en.wikipedia.org/wiki/Hayao_Miyazaki) is written in English with given name first and with a space between the two name fields, while in Japanese there is no space with surname first: 
+Some writing systems require spaces (or some other non-letters) to separate words. For example, [Hayao Miyazaki](https://en.wikipedia.org/wiki/Hayao_Miyazaki) is written in English with given name first and with a space between the two name fields, while in Japanese there is no space with surname first: [宮崎駿](https://ja.wikipedia.org/wiki/%E5%AE%AE%E5%B4%8E%E9%A7%BF)
 
-> [宮崎駿](https://ja.wikipedia.org/wiki/%E5%AE%AE%E5%B4%8E%E9%A7%BF)
+Those locales are determined using the Script Metadata as follows.
+1. Get the likely script of the formatting locale, using LikelySubtags
+2. If the likely script is Thai, it needs spaces.
+3. Otherwise, if the likely script is Jpan, Hant, or Hans, it doesn't need spaces
+4. Otherwise, if the likely script has the script metadata property lbLetters = YES, it doesn't need spaces
+5. Otherwise, it needs spaces
+
+Then:
 
 1. If a locale requires spaces between words, the normal patterns for the formatting locale are used. On Wikipedia, for example, note the space within the Japanese name on pages from English and Korean (an ideographic space is used here for emphasis).
 
 * “​​[Hayao Miyazaki (宮崎<span style="background-color:aqua">　</span>駿, Miyazaki Hayao](https://en.wikipedia.org/wiki/Hayao_Miyazaki)…” or 
 * “[미야자키<span style="background-color:aqua">　</span>하야오(일본어: 宮﨑<span style="background-color:aqua">　</span>駿 Miyazaki Hayao](https://ko.wikipedia.org/wiki/%EB%AF%B8%EC%95%BC%EC%9E%90%ED%82%A4_%ED%95%98%EC%95%BC%EC%98%A4)…”. 
 
-2. If a locale **doesn’t** require spaces between words, there are two cases, based on whether the foreign name is written in the locale’s script, or the foreign name is left in its native script. In both cases, patterns from the **locale of the name** are used. For example, the formatting locale might be Japanese, and the locale of the PersonName object might be de_CH, German (Switzerland), such as Albert Einstein.
+2. If a locale **doesn’t** require spaces between words, there are two cases, based on whether the name is foreign or not (based on the PersonName objects explicit or calculated locale's language subtag). For example, the formatting locale might be Japanese, and the locale of the PersonName object might be de_CH, German (Switzerland), such as Albert Einstein.
 
-    1. **The foreign name is written in the formatting locale’s script.** In that case, the **foreignSpaceReplacement** is substituted for each space in the patterns from the _locale of the name_. Here are examples for Albert Einstein in Japanese and Chinese:
+    1. **The name is native** In that case, the patterns are used as is.
```

Comment:
What happened to `nativeSpaceReplacement`?

Also, I think the way I implemented cross-script formatting differs a bit from what you have here.

I think what I've got right now is:
1. If the formatter locale uses spaces, don't do anything special.
2. If the formatter locale doesn't use spaces, the name locale does use spaces, and the actual text of the name is in a script that is not native to the formatter locale, use the name locale's formatting patterns instead of the formatter locale's patterns.
3. Otherwise, use the formatter's patterns to format the name.  Afterwards, if the name locale and formatter locale agree, apply the `nativeSpaceReplacement`, and if they don't, apply the `foreignSpaceReplacement`.